app/themes/base.js

```diff
@@ -0,0 +1,82 @@
+import {
+  teal500,
+  teal600,
+  teal700,
+  white,
+  darkBlack,
+  deepPurple500,
+  deepPurple600,
+  deepPurple700,
+  grey50,
+  grey100,
+  grey200,
+  grey300,
+  grey400,
+  grey500,
+  grey600,
+  grey700,
+  grey800,
+  grey900,
+  fullBlack
+} from 'material-ui/styles/colors'
+
+export default {
+  breakpoints: {
+    mobile: '13.6em',
+    mobileWide: '30.4em',
+    tablet: '47.2em',
+    desktop: '64em',
+    desktopLarge: '97.6em'
+  },
+  fontSize: '15px',
+  fontSize1: '.875rem',
+  fontSize2: '1rem',
+  fontSize3: '1.25rem',
+  fontSize4: '1.5rem',
+  fontSize5: '2.25rem',
+  fontSize6: '3rem',
+  fontSize7: '5rem',
+  fontSize8: '6rem',
+  fontFamily: 'Roboto, sans-serif',
+  weightThin: 100,
+  weightLight: 300,
+  weightRegular: 400,
+  weightMedium: 500,
+  weightBold: 700,
+  weightBlack: 900,
+  em: '1em',
+  rem: '1rem',
+  borderRadius: '2px',
+  zIndexMax: 1000,
+  primary1Color: teal500,
```

Comment:
now looking at this, i reckon the color keys should be `colorPrimary1`, `colorText`, etc. since everything else is done this way.